### ISSUE TITLE
Group content categories

### ISSUE THREAD

**Issue by gavinr** [Esri/arcgis-rest-js#676]
Similar to [`getGroupContent`](https://esri.github.io/arcgis-rest-js/api/portal/getGroupContent/) and [`getGroupUsers`](https://esri.github.io/arcgis-rest-js/api/portal/getGroupUsers/), does it make sense to add `getGroupCategorySchema` ([doc](https://developers.arcgis.com/rest/users-groups-and-items/group-category-schema.htm))?

(and perhaps related [assign](https://developers.arcgis.com/rest/users-groups-and-items/assign-group-category-schema.htm), [delete](https://developers.arcgis.com/rest/users-groups-and-items/delete-group-category-schema.htm), and [update](https://developers.arcgis.com/rest/users-groups-and-items/update-group-items-with-content-categories.htm))

**Comment by MikeTschudi** [comment]
@gavinr, your wish is my command: PR #682

**Comment by gavinr** [comment]
@MikeTschudi yes!! Thanks! Looking forward to testing it out once it gets pushed to a release. 🎉 🎉

**Comment by gavinr-maps** [comment]
Closed in https://github.com/Esri/arcgis-rest-js/pull/682
